### PR TITLE
Renaming DACCS to Marble in service-config.json.template files

### PR DESCRIPTION
**Pull request by dchandan**
## Overview

Small change to the location of schema describing node services following the renaming of DACCS to Marble.

## Changes

**Non-breaking changes**
- Changed https://github.com/DACCS-Climate/DACCS-node-registry to https://github.com/DACCS-Climate/Marble-node-registry in all `service-config.json.template` files.

**Breaking changes**
None

### CODE REVIEW

File: tests/test_deployment.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import os
-import pytest
 import glob
 import json
+import os
```
Comment (tlvu) [inline tests/test_deployment.py:3]: I am surprised the import is really missing?

**Review by tlvu — APPROVED** [review]
LGTM but I think @mishaschwartz knows more about this rename.

File: tests/test_deployment.py (hunk 1 above)
Comment (dchandan) [reply tests/test_deployment.py:3]: No it's not, its just the IDE sorting the imports.

**Review by mishaschwartz — APPROVED** [review]
Looks good. ~This makes me realize that we need a servic-config.json.template file for stac as well. Though we should do that in a different PR.~

EDIT: @dchandan you're on it already! (#373) Thank you!

**Review by fmigneault — APPROVED** [review]

**Pull request merged**